server/src/controllers/AlumniController.js

```diff
@@ -35,15 +35,18 @@ module.exports = {
       await Alumnus.create(alumnus);
       // console.log(req.body);
       res.status(200).json({ Status: "OK" });
+      next();
     } catch(e){
       return res.status(500).json({ error: e.message });
     }
 
   },
-  async allAlumni(_,res){
+  async allAlumni(_,res, next){
     try{
       const alumni = await Alumnus.find({});
-      return res.status(200).json({ alumni });
+      res.cache = alumni;
```

Comment:
Can't we skip adding the 'cache'?  
Can't the middleware read off the value directly from the body?

Reply:
Could you please cite an example for this? The body is what I am passing down through the res.cache, the [response object](http://expressjs.com/en/4x/api.html#res) doesn't have a `body` field, so we cannot get the generated body from there.

Reply:
Have a look at this: http://expressjs.com/en/4x/api.html#res.get